From: David Bauer <mail@david-bauer.net>
Date: Mon, 21 Aug 2023 23:16:14 +0200
Subject: ath79: don't create DIR-825 B1 factory image

Currently the build fails for the D-Link DIR-825 B1. THis is due to the
factory image being size-constrained. The sysupgrade image is not
affected, as OpenWrt now uses a concatenated firmware partition.

To newly install such a device, please use the latest OpenWrt 22.03
factory image and install a Gluon sysupgrade.

Signed-off-by: David Bauer <mail@david-bauer.net>

diff --git a/target/linux/ath79/image/generic.mk b/target/linux/ath79/image/generic.mk
index 68c410556151304f8fb5e7b6df57c17363fa3508..c65c1155c2d0d2550fa1162ac5fbe3523a1e155d 100644
--- a/target/linux/ath79/image/generic.mk
+++ b/target/linux/ath79/image/generic.mk
@@ -1196,11 +1196,6 @@ define Device/dlink_dir-825-b1
   DEVICE_PACKAGES := kmod-usb-ohci kmod-usb2 kmod-usb-ledtrig-usbport \
 	kmod-leds-reset kmod-owl-loader kmod-switch-rtl8366s
   IMAGE_SIZE := 7808k
-  FACTORY_SIZE := 6144k
-  IMAGES += factory.bin
-  IMAGE/factory.bin = append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | \
-	pad-rootfs | check-size $$$$(FACTORY_SIZE) | pad-to $$$$(FACTORY_SIZE) | \
-	append-string 01AP94-AR7161-RT-080619-00
 endef
 TARGET_DEVICES += dlink_dir-825-b1
 
